policyengine_canada/parameters/gov/provinces/mb/tax/income/credits/eligible_dependant_amount/dependant_income_max_amount.yaml

```diff
@@ -0,0 +1,11 @@
+description: Manitoba provides tax credit to eligible dependants whose annual net income is less than a certain amount.
+values:
+  2023-01-01: 9_134
+metadata:
+  unit: currency-CAD
+  label: Manitoba dependent's maxiumum eligible annual net income
+  reference:
+    - title: 2023 Manitoba Personal Tax Credits Return
+      href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1mb/td1mb-23e.pdf
+    - title: Government of Canada - Manitoba individual income tax C.C.S.M C.l.10 Division II Section4.6(6)(i)
+      href: https://web2.gov.mb.ca/laws/statutes/archive/i010(2020-11-05)e.php#4.6(6)
```

Comment:
Please cite the legal code even if the parameters are outdated due to inflation adjustments, to indicate the legal structure of the program and how the parameter fits in